### ISSUE TITLE
Some doubts about your programs

### ISSUE THREAD

**Issue by Ostnie** [jeong-tae/RACNN-pytorch#1]
I know nothing about pytorch, so I read your programs as tensorflow, and I find a strange calculations about loss, in model/loss.py, I see you cal loss as  
F.cross_entropy(preds[i], labels)
Is this right? I guess maybe F.cross_entropy(preds[i], labels**[i]**) ?

**Comment by jeong-tae** [comment]
the 'i' in https://github.com/jeong-tae/RACNN-pytorch/blob/master/models/Loss.py#L7 is for the each scales. I currently use 3 different scales. so 'i' is range in (0, 3)
labels are same for each scales.

**Comment by Ostnie** [comment]
You know, I still have some doubt about backpropagation for APN, I think it is incorrect to use a model's result as another model's loss,  all the parameters should be calculated by the chain rule in backwardpropagation and Lrank_loss does not match this. 

But in formula 11
![image](https://user-images.githubusercontent.com/31840090/40721421-0bf9411a-644c-11e8-9a9a-20ecf8ffabee.png)
I think there is no way to calculate the left side of the formula, but the right side of the formula can be calculated.  You said that your apn does not converging well, and I didn't find a conversion method like formula 11 in your program. I don't know if this is the cause of the problem, this is just a guess .

**Comment by jeong-tae** [comment]
i will close this issue. if you still have any question, re-open or make a new issue to contribute.
i will wait a good news to improve current repository. thx
